lib/utils.pm

```diff
@@ -1057,6 +1058,23 @@ sub is_boot_encrypted {
     return 1;
 }
 
+=head2 need_passphrase_again_in_grub
```

Comment:
There is already `workaround_type_encrypted_passphrase`, but it needs to be used correctly

Reply:
Thanks, I will double check it!

Reply:
> I would like to keep the current logic

That's what my comment on the progress ticket is about: The logic that's currently in the test code is completely backwards and needs to be ripped out and replaced.